src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -240,7 +240,7 @@ public static partial class JsonSerializer
         public static TValue? Deserialize<TValue>(this System.Text.Json.Nodes.JsonNode? node, System.Text.Json.JsonSerializerOptions? options = null) { throw null; }
```

Comment:
this is a reminder to me to always use the tool to update these ref files, as with the churn here it's hard to see what actually changed 😄

Reply:
😅 I reverted extraneous diffs to make the actual changes clearer. Can follow up with a dedicated PR to clean up the ref file.